_includes/install.md

```diff
@@ -20,17 +26,24 @@ And then install it with
 npm install -g <PATH_TO_FILE>/effekt.tgz
 ```
 
+#### Install location
+
 You can find the installation location by running
 
 ```bash
-npm root -g
+$ npm config get prefix
```

Comment:
the `$` is inconsistent now with the rest of the `bash` excerpts

Reply:
I have Effekt install via `sbt install` which in turn uses `npm install -g`, yet

```
$ npm root -g
/opt/homebrew/lib/node_modules

$ npm config get prefix
/opt/homebrew
```

while

```
$ which effekt
/opt/homebrew/bin/effekt
```

Hence, this should work in any case by just including more directories in the `$PATH`.

Reply:
The file in `bin` is just a symlink, but the bigger problem is that the `npm root -g` can be user configured to be a different folder outside of the npm prefix.

Reply:
Good point, thanks. I'll change it again.

> [...] but the bigger problem is that the npm root -g can be user configured to be a different folder outside of the npm prefix.

Well, we cannot account for every eventuality.